common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1744,7 +1758,14 @@ object PageElement {
     }
   }
 
-  private[pageElements] def containsThirdPartyTracking(embedTracking: Option[EmbedTracking]): Boolean = {
+  private[pageElements] def containsThirdPartyTracking(
+      embedTracking: Option[EmbedTracking],
+      isLiveblog: Boolean,
+  ): Boolean = {
+    if (isLiveblog) {
+      return true
+    }
```

Comment:
This will add the overlay to _all_ embed element in liveblogs. Have we got sign off from editorial on this?

Reply:
I think its fairly rare to see embeds in Liveblogs except for Twitter embeds, but we don't actually ever apply a click to view wrapper to those in DCR and this change won't affect them. I can inform Editorial about this change but my gut feeling is that this is something we have to do.

Reply:
> Aside: is this change temporary pending a fix in CAPI?

It is temporary yep, I've raised the issue with CAPI and its on their radar now, I'm not too sure when they'll have time to do it though.